src/Framework/PaymentGateways/Routes/GatewayRoute.php

```diff
@@ -2,74 +2,90 @@
 
 namespace Give\Framework\PaymentGateways\Routes;
 
-use Give\Framework\PaymentGateways\Contracts\OffsiteGatewayInterface;
 use Give\Framework\PaymentGateways\DataTransferObjects\GatewayRouteData;
 use Give\Framework\PaymentGateways\Exceptions\PaymentGatewayException;
 use Give\Framework\PaymentGateways\PaymentGateway;
 use Give\Framework\PaymentGateways\PaymentGatewayRegister;
 
+use function Give\Framework\Http\Response\response;
+
 /**
  * @since 2.18.0
  */
 class GatewayRoute
 {
     /**
-     * @since 2.18.0
+     * This is our entry point into the Gateway Routing system.
      *
      * @return void
      * @throws PaymentGatewayException
+     * @since 2.18.0
+     * @unreleased - validate secureRouteMethods
```

Comment:
Should be
- Description
- `@since`
- `@param`
- `@return`
- `@throws`

Reply:
@jonwaldstein This is a list of PHPDoc tags in PHPStorm.

![Preferences 2022-02-01 at 8 16 45 AM](https://user-images.githubusercontent.com/1784821/151906027-b6c415e9-9733-4590-980c-4674362a2402.jpg)

Reply:
My editor shuffled these around. Is this something that's in the coding standards repo?  Could be a setting that didn't make it after getting the new mac.

Reply:
We need to generate an `.editorConfig` that contains everything. For now, you have to set those manually in PhpStorm.

Reply:
That would be convenient